dora/tests/src/test/java/alluxio/server/worker/WorkerMetadataSyncIntegrationTest.java

```diff
@@ -44,7 +45,9 @@ public class WorkerMetadataSyncIntegrationTest {
                   .build(),
           Configuration.modifiableGlobal());
 
+  /* Not applied as registration is not going thru master any more */
```

Comment:
delete irrelevant test case and same for below?

Reply:
i will track deleting irrelevant test bcos of registration workflow change as a separate effort, as there is A LOT